src/constraint_solver/io.cc

```diff
@@ -2291,13 +2310,28 @@ bool CpModelLoader::BuildFromProto(const CpSequenceVariable& proto) {
   return true;
 }
 
+size_t CpModelLoader::IntegerExpressionCount() const {
+  return expressions_.size();    
+}
+
 IntExpr* CpModelLoader::IntegerExpression(int index) const {
   CHECK_GE(index, 0);
   CHECK_LT(index, expressions_.size());
   CHECK(expressions_[index] != nullptr);
   return expressions_[index];
 }
 
+IntExpr* CpModelLoader::IntegerExpressionByName(const std::string& name) const {
```

Comment:
@lperron Name is already being stored. That's what ``SetName`` and ``Name`` buy us. If a client doesn't use this API, great. But when we do want to use it, it will be there. Should be transparent to either interest.